Sources/Datadog/RUM/Instrumentation/Actions/SwiftUI/SwiftUIActionModifier.swift

```diff
@@ -51,6 +51,48 @@ public extension SwiftUI.View {
     }
 }
 
+#else
+
+/// `SwiftUI.ViewModifier` for RUM which invoke `addUserAction` from the
+/// global RUM Monitor when the modified view receives a click from the remote control.
+@available(tvOS 14, *)
+internal struct RUMClickActionModifier: SwiftUI.ViewModifier {
+    /// The minimum duration of the long press that must elapse before the gesture succeeds.
+    let minimumDuration: Double
+
+    /// Action Name used for RUM Explorer.
+    let name: String
+
+    /// Custom attributes to attach to the Action.
+    let attributes: [AttributeKey: AttributeValue]
+
+    func body(content: Content) -> some View {
+        content.simultaneousGesture(
+            LongPressGesture(minimumDuration: minimumDuration).onEnded { _ in
+                Global.rum.addUserAction(type: .click, name: name, attributes: attributes)
+            }
+        )
+    }
+}
+
+@available(tvOS 14, *)
+public extension SwiftUI.View {
+    /// Monitor this view tap actions with Datadog RUM. An Action event will be logged after the minimum duration of presss.
+    ///
+    /// - Parameters:
+    ///   - name: The action name.
+    ///   - attributes: custom attributes to attach to the View.
+    ///   - minimumDuration: The minimum duration of the long press that must elapse to complete the click action.
+    /// - Returns: This view after applying a `ViewModifier` for monitoring the view.
+    func trackRUMClickAction(
+        name: String,
+        attributes: [AttributeKey: AttributeValue] = [:],
+        minimumDuration: Double = 0.01
```

Comment:
`TimeInterval` (which is a typealias to `Double`) could be more relevant here, WDYT? Also, adding "(in seconds)" to the `minimumDuration` comment could be handy.

Reply:
Indeed, I need to specify that in the doc 👍 
For the duration type, I agree it would be more relevant! I've based this signature on the type used in [LongPressGesture](https://developer.apple.com/documentation/swiftui/longpressgesture) which use `Double`.